taxrate.go

```diff
@@ -25,13 +25,11 @@ type TaxRatePercentageRangeQueryParams struct {
 // TaxRateListParams is the set of parameters that can be used when listing tax rates.
 // For more detail see https://stripe.com/docs/api/tax_rates/list.
 type TaxRateListParams struct {
-	ListParams      `form:"*"`
-	Active          *bool                              `form:"active"`
-	Created         *int64                             `form:"created"`
-	CreatedRange    *RangeQueryParams                  `form:"created"`
-	Inclusive       *bool                              `form:"inclusive"`
-	Percentage      *float64                           `form:"percentage"`
-	PercentageRange *TaxRatePercentageRangeQueryParams `form:"percentage"`
```

Comment:
Can/should we delete this `TaxRatePercentageRangeQueryParams` struct b/c this is the only reference?